playbooks/ka-init/group_vars/all.yml

```diff
@@ -93,6 +99,7 @@ kubectl_proxy_port: 8088
 # ovn vars.
 # ----------------------------
 ovn_image_repo: "docker.io/ovnkube/ovn-daemonset-u:latest"
+enable_ovn_raft: True
```

Comment:
Currently default kube-ansible set up single cluster (!= HA), so I'm just wondering it works or not.

How about to make it comment?

Reply:
oh yeah, my bad. It should be commented out.